docs/self-hosted/self-hosted.md

```diff
@@ -10,9 +10,11 @@ Gitpod, just as you know it from [gitpod.io](https://gitpod.io), can be deployed
 
 ## Installation
 
-You can find all configuration templates and installation scripts in the Gitpod repository:
+You can find all the installation methods and relevant scripts under the [`install`](https://github.com/gitpod-io/gitpod/tree/master/install) directory.
```

Comment:
This is the beginning of the funnel and here I thing we should not guide the users away from the docs (and into the source code). People should read the docs first and after they did it we can tell them "for further reading, see here is the source code and more READMEs".